src/assets/layouts/personaldata.scss

```
@@ -0,0 +1,81 @@
+@import "../var";
+.personal{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+    &__aside{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 64px;
+        height: 100vh;
+        background: rgba(196, 196, 196, 0.35);
+    }
+    &__info{
+        margin-bottom: 26px;
+    }
+    &__avatar{
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        &--edit{
+            justify-content: space-between;
+        }
+        &-upload{
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            background-color: #8A4CA7;
+            padding: 8px 40px;
+            border-radius: 6px;
+            color: #ffffff;
+            font-size: 16px;
+            text-decoration: none;
+            min-height: 40px;
+        
+        }
+    }
+    &__nick{
+        text-align: center;
+        font-weight: 700;
+        margin-bottom: 26px;
+        font-size: 16px;
+    }
+    &__backlink{
+        display: flex;
+        transform: rotate(-180deg);
+    }
+    &__wrap{
+        margin: 0 auto;
+        width: 400px;
+    }
+    &__info{
+        display: flex;
+        flex-direction: column;
+    }
+    &__form{
+        &-field{
+            margin-bottom: 11px;
+        }
+    }
+    &__actions{
+        display: flex;
```

Между селекторами принято оставлять пустую строку:

```suggestion
    &__form{
        &-field{
            margin-bottom: 11px;
        }
    }
    
    &__actions{
        display: flex;
```

---

Подключу prettier во втором спринте, сейчас отредактировал при помощи vscode